GitUI/BranchTreePanel/RepoObjectsTree.cs

```diff
@@ -278,6 +280,7 @@ public void SelectionChanged(IReadOnlyList<GitRevision> selectedRevisions)
             ThreadHelper.JoinableTaskFactory.RunAsync(async () =>
             {
                 cancellationToken.ThrowIfCancellationRequested();
+                await TaskScheduler.Default;
```

Comment:
❔ Why are we adding this? The call to `GetMergedBranchesAsync` should be properly asynchronous in the case where it's used, and for other cases we're just going to switch right back to the main thread. The previous code was more efficient.

Reply:
Git Command Log reports that the command executes on the UI thread
The call is like `.GetOutputAsync(GitCommand).ConfigureAwait(false))`

Reply:
See https://github.com/gitextensions/gitextensions/pull/8851/files#r575733576